ros/install.bash

```diff
@@ -19,7 +19,7 @@ rosdep_update_file="/tmp/tue_rosdep_update_${USER}"
 if [ ! -f "$rosdep_update_file" ]
 then
     tue-install-debug "Updating rosdep"
-    tue-install-pipe rosdep update
+    tue-install-pipe rosdep update --rosdistro "$TUE_ROS_DISTRO"
```

Comment:
Actually with tue-env, update of rosdep isn't needed at all as we are not using it for anything

Reply:
Which reminds me. We solve all unfound targets using rosdep. Still on my to-backport list

Reply:
But the image recognition packages use catkin lint. Which uses rodeo. Therefore it needs to be updated.